contents/docs/integrate/_snippets/install-nuxt.mdx

```diff
@@ -24,30 +24,20 @@ export default defineNuxtConfig({
 3. Create a new plugin by creating a new file `posthog.client.js` in your [plugins directory](https://nuxt.com/docs/guide/directory-structure/plugins).
 
 ```js file=plugins/posthog.client.js
-import { defineNuxtPlugin, useRuntimeConfig, useRouter, nextTick } from '#imports'
+import { defineNuxtPlugin, useRuntimeConfig } from '#imports'
 
 import posthog from 'posthog-js'
 export default defineNuxtPlugin(() => {
     const runtimeConfig = useRuntimeConfig()
     const posthogClient = posthog.init(runtimeConfig.public.posthogPublicKey, {
         api_host: runtimeConfig.public.posthogHost,
-        capture_pageview: false, // we add manual pageview capturing below
+        capture_pageview: 'history_change',
```

Comment:
Add a comment explaining that this will automatically track `$pageview`s on router changes
Have you tested whether hooking the navigator works for nuxt?